src/app/(pages)/[slug]/page.tsx

```diff
@@ -1,89 +1,109 @@
-import { fetchDoc } from '../../_api/fetchDoc'
-import { fetchDocs } from '../../_api/fetchDocs'
-import { generateMeta } from '../../_utilities/generateMeta'
-import { staticHome } from './staticHome'
+import { Metadata } from "next";
+import { draftMode } from "next/headers";
+
+
+import { Page } from "../../../payload/payload-types";
+import { fetchDoc } from "../../_api/fetchDoc";
+
+import { generateMeta } from "../../_utilities/generateMeta";
+
+import styles from "./styles.module.css";
+
+
 import { Subscribe } from "@/app/_blocks/Subscribe";
+import { ALL_CONTENT } from "@/app/_graphql/allContent";
+import ContentCard from "@/app/_components/ContentCard";
+import { fetcher } from "@/app/_utilities/fetcher";
 
 // Payload Cloud caches all files through Cloudflare, so we don't need Next.js to cache them as well
 // This means that we can turn off Next.js data caching and instead rely solely on the Cloudflare CDN
 // To do this, we include the `no-cache` header on the fetch requests used to get the data for this page
 // But we also need to force Next.js to dynamically render this page on each request for preview mode to work
 // See https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic
 // If you are not using Payload Cloud then this line can be removed, see `../../../README.md#cache`
-export const dynamic = 'force-dynamic'
+export const dynamic = "force-dynamic";
 
-export default async function Page({ params: { slug = 'home' } }) {
-  const { isEnabled: isDraftMode } = draftMode()
 
-  let page: Page | null = null
+export default async function Page({ params: { slug = "home" } }) {
+  const content = await fetcher({ query: ALL_CONTENT });
 
-  try {
-    page = await fetchDoc<Page>({
-      collection: 'pages',
-      slug,
-      draft: isDraftMode,
-    })
-  } catch (error) {
-    // when deploying this template on Payload Cloud, this page needs to build before the APIs are live
-    // so swallow the error here and simply render the page with fallback data where necessary
-    // in production you may want to redirect to a 404  page or at least log the error somewhere
-    // console.error(error)
-  }
+  const content_list = Object.keys(content).flatMap(key =>
+    content[key].map(content => ({
+      contentType: key,
+      content: content,
+    })));
 
-  // if no `home` page exists, render a static one using dummy content
-  // you should delete this code once you have a home page in the CMS
-  // this is really only useful for those who are demoing this template
-  if (!page && slug === 'home') {
-    page = staticHome
-  }
 
-  if (!page) {
-    return notFound()
-  }
+  return (
+    <>
+      {/* Head Block*/}
+      <div className={styles.headBlock}>
+        <div className={styles.imageContainer}>
+          <img className={styles.topImage} src={"/contenthub-top-img.png"} />
+        </div>
 
-  const { hero, layout } = page
+        {/*  Top Highlight  */}
```

Comment:
These highlights could be a separate component, somewhere down the line

Reply:
yep, after #40 is done it's the next thing on the list